### ISSUE TITLE
continuous_output=True prints bytes instead of strings in Python3

### ISSUE THREAD

**Issue by nicholasbishop** [AmadeusITGroup/JumpSSH#23]
Example output:

```
DEBUG:jumpssh.session:Running command 'sudo yum install -y python36' on '192.168.39.110' as ec2-user...
b'Loaded plugins: priorities, update-motd, upgrade-helper\r\n'
b'Package python36-3.6.2-5.7.amzn1.x86_64 already installed and latest version\r\nNothing to do\r\n'
```

**Comment by t-cas** [comment]
Thanks @nicholasbishop for the report of the issue, I will provide a fix shortly, this is quite easy to fix

**Comment by t-cas** [comment]
fix delivered in release 1.3.2

**Comment by nicholasbishop** [comment]
Thanks :)
